trigger.js

```diff
@@ -139,11 +139,19 @@ class TriggerHappy {
     _isSceneTrigger(scene, trigger) {
         return trigger.trigger.entity === "Scene" && trigger.trigger.id === scene.id;
     }
+
+    _tokenContains(token, position) {
+        if (Number.between(position.x, token.data.x, token.data.x + token.w)
+        && Number.between(position.y, token.data.y, token.data.y + token.h)) {
+        console.log(position.x, token.data.x, token.data.x + token.w, Number.between(position.x, token.data.x, token.data.x + token.w));
+        console.log(position.y, token.data.y, token.data.y + token.h, Number.between(position.y, token.data.y, token.data.y + token.h))
+        }
+        return  Number.between(position.x, token.data.x, token.data.x + token.w)

```

Comment:
I didn't know about `Number.between` 👍 
There's a an unnecessary `console.log` above which I assume you forgot.
also, I prefer to indent split lines, so the `&&` in the line below should be indented once more

Reply:
> Humm.. I'll do a quick review of this, but I feel like it might be adding too much complexity perhaps? It might be better to just make the users place multiple tokens, it's much easier for them to copy/paste and put 9 tiles than to have complex code handling that use case. Just needs to be written in the documentation.

Except that by placing 9 tokens you would need 9 trigger entries in trigger happy and would end up checking 9 tokens for intersections on every move, which is not a good return on cpu investment.